js_modules/dagster-ui/packages/ui-core/src/assets/AssetsOverview.tsx

```diff
@@ -306,85 +300,93 @@ export const AssetsOverview = ({viewerName}: {viewerName?: string}) => {
                   <CountForAssetType key={computeKind} assetsCount={assetCount}>
                     <TagIcon label={computeKind} />
-                    <Link to={linkToAssetGraphComputeKind(computeKind)}>{computeKind}</Link>
+                    <Link to={linkToAssetGraphComputeKind(computeKind)} style={TextOverflowStyle}>
+                      {computeKind}
+                    </Link>
                   </CountForAssetType>
                 ))}
-              </SectionBody>
+              </GridStyle>
             </>
           )}
           {assetCountBySection.countPerAssetGroup.length > 0 && (
             <>
               <SectionHeader sectionName="Asset groups" />
-              <SectionBody>
+              <GridStyle>
                 {assetCountBySection.countPerAssetGroup.map((assetGroupCount) => (
                   <CountForAssetType
                     key={JSON.stringify(assetGroupCount.groupMetadata)}
                     assetsCount={assetGroupCount.assetCount}
                   >
                     <Icon name="asset_group" />
-                    <Link to={linkToAssetGraphGroup(assetGroupCount.groupMetadata)}>
+                    <Link
+                      to={linkToAssetGraphGroup(assetGroupCount.groupMetadata)}
+                      style={TextOverflowStyle}
+                    >
                       {assetGroupCount.groupMetadata.groupName}
                     </Link>
-                    <span style={{color: Colors.textLighter()}}>
+                    <span style={{...TextOverflowStyle, ...{color: Colors.textLighter()}}}>
                       {repoAddressAsHumanString({
                         name: assetGroupCount.groupMetadata.repositoryName,
                         location: assetGroupCount.groupMetadata.repositoryLocationName,
                       })}
                     </span>
                   </CountForAssetType>
                 ))}
-              </SectionBody>
+              </GridStyle>
             </>
           )}
           {assetCountBySection.countPerCodeLocation.length > 0 && (
             <>
               <SectionHeader sectionName="Code locations" />
-              <SectionBody>
+              <GridStyle>
                 {assetCountBySection.countPerCodeLocation.map((countPerCodeLocation) => (
                   <CountForAssetType
                     key={repoAddressAsHumanString(countPerCodeLocation.repoAddress)}
                     assetsCount={countPerCodeLocation.assetCount}
                   >
                     <Icon name="folder" />
-                    <Link to={linkToCodeLocation(countPerCodeLocation.repoAddress)}>
+                    <Link
+                      to={linkToCodeLocation(countPerCodeLocation.repoAddress)}
+                      style={TextOverflowStyle}
+                    >
                       {repoAddressAsHumanString(countPerCodeLocation.repoAddress)}
```

Comment:
I believe you want to <MiddleTruncate> this text

Reply:
ahh good call, changed